book/src/kubernetes-changelog.md

```diff
@@ -3,6 +3,42 @@
 This page summarizes major CSI changes made in each Kubernetes release. For
 details on individual features, visit the [Features section](features.md).
 
+## Kubernetes 1.26
+
+### Features
+* GA
+    * Delegate fsgroup to CSI driver
+    * Azure File CSI migration
+    * vSphere CSI migration
+* Alpha
+    * Cross namespace volume provisioning
+
+## Kubernetes 1.25
+
+### Features
+* GA
+    * CSI inline volume
+    * Core CSI migration
+    * AWS EBS CSI migration
+    * GCE PD CSI migration
+* Beta
+    * vSphere CSI Migration (on by default)
+    * Portworx CSI Migration (off-by-default)
+
+## Kubernetes 1.24
+
+### Features
+* GA
+    * Volume expansion
+    * CSI storage capacity tracking
```

Comment:
just "Storage capacity tracking" to match the previous entries

Reply:
Done